src/afrolabs/components/kafka.clj

```diff
@@ -1200,6 +1226,8 @@
       (handle-consumption-results
           [this msgs]
 
+        (log/warn "Using a producer component's `IConsumedResultsHandler` implementation is now considered harmful.")
+
```

Comment:
no, just spamming logs for now. Once I've run this in prod and haven't seen these messages I'll remove the fn from the protocol. This would be a breaking change in a long line of not making breaking changes, but I guess that's ok...